requirements.in

```diff
@@ -1,4 +1,4 @@
 pip-tools
 django
 djangorestframework
-mysqlclient
+mysqlclient
```

Comment:
dodaj pustą linie we wszystkich plikach tam gdzie wyświetla się ta czerwona ikonka